src/js/components/AppListComponent.jsx

```diff
@@ -455,6 +456,27 @@ var AppListComponent = React.createClass({
 
     var totalColumnSpan = 8;
 
+    var statusDocsURL = "https://mesosphere.github.io/marathon/docs/" +
+      "marathon-ui.html#application-status-reference";
+    var statusTooltipMessage = (
+      <span>
+        At-a-glance overview of the global application or group state.
+        Includes Running, Deploying, Suspended, Delayed and Waiting.
```

Comment:
There's no meaningful space between these lines which means that the text renders as `...Delayed and Waiting.Read More.`. Let's fix this with margins.